Makefile.am

```diff
@@ -230,9 +230,9 @@ rpmvar_DATA =
 
 install-exec-hook:
 	@rm -f $(DESTDIR)$(bindir)/rpmquery
-	@LN_S@ ../../bin/rpm $(DESTDIR)$(bindir)/rpmquery
+	@LN_S@ rpm $(DESTDIR)$(bindir)/rpmquery
```

Comment:
The prefix is in general user-configurable.  On my system, it's `/usr/local/bin/`, and so the symlink points to `/usr/bin/rpm`, which does not exist.

I now understand the reason for why this is done this way.  But perhaps this should either be a configure option or arranged as a post-installation fixup in the spec file.